# Description: Trivial patch to add main category to desktop file, it fixes a
#  lintian warning.
# Author: José Manuel Santamaría Lema
# Forwarded: No, it's fixed in upstream's svn.
--- krecipes-2.0~alpha1.orig/src/krecipes.desktop
+++ krecipes-2.0~alpha1/src/krecipes.desktop
@@ -10,7 +10,7 @@
 Exec=krecipes %i -caption "%c"
 Icon=krecipes
 Type=Application
-Categories=Qt;KDE;Database;X-KDE-More;
+Categories=Utility;Qt;KDE;Database;X-KDE-More;
 X-DocPath=krecipes/index.html
 GenericName=Cooking Book
 GenericName[be]=Кніга рэцэптаў
